surfactant/sbomtypes/_system.py

```diff
@@ -21,3 +22,36 @@ class System:
     vendor: Optional[List[str]] = None
     description: Optional[str] = None
     provenance: Optional[List[SystemProvenance]] = None
+
+    def merge(self, sy: System):
```

Comment:
Ah, forgot member function type annotations referring to themselves are kinda weird. I think adding `from __future__ import annotations` to the top of the imports might allow this (Python 3.7+ feature, which is fine since Surfactant requires Python 3.8+).